### PR TITLE
Add System.Memory and enable new ToCamelCase code

### PR DESCRIPTION
**Pull request by Shane32**

### CODE REVIEW

**Comment by Shane32** [comment]
@sungam3r ok?

**Comment by Shane32** [comment]
This test keeps breaking CI runs:
image

I'll re-run tests so hopefully they pass.

File: src/GraphQL/StringExtensions.cs [hunk 1]
```diff
@@ -29,9 +31,9 @@ public static string ToCamelCase(this string s)
             if (newFirstLetter == s[0])
                 return s;
 
-            return //s.Length <= 256
-                //? FastChangeFirstLetter(newFirstLetter, s)
-                //:
+            return s.Length <= 256
+                ? FastChangeFirstLetter(newFirstLetter, s)
+                :
                 newFirstLetter + s.Substring(1);
```
Comment (sungam3r) [inline src/GraphQL/StringExtensions.cs:37]: Move this on previous line.

File: src/GraphQL/StringExtensions.cs [hunk 2]
```diff
@@ -51,18 +53,18 @@ public static string ToPascalCase(this string s)
             if (newFirstLetter == s[0])
                 return s;
 
-            return //s.Length <= 256
-               //? FastChangeFirstLetter(newFirstLetter, s)
-               //:
+            return s.Length <= 256
+               ? FastChangeFirstLetter(newFirstLetter, s)
+               :
                newFirstLetter + s.Substring(1);
```
Comment (sungam3r) [inline src/GraphQL/StringExtensions.cs:59]: Here too.

File: src/GraphQL/StringExtensions.cs (hunk 1 above)
Comment (Shane32) [inline src/GraphQL/StringExtensions.cs:37]:
```suggestion
                : newFirstLetter + s.Substring(1);
```

File: src/GraphQL/StringExtensions.cs (hunk 2 above)
Comment (Shane32) [inline src/GraphQL/StringExtensions.cs:59]:
```suggestion
               : newFirstLetter + s.Substring(1);
```

**Pull request merged**